api/XMLDocument.json

```diff
@@ -23,22 +37,43 @@
             "version_added": false
           },
           "opera": {
-            "version_added": false
+            "version_added": "21"
           },
           "opera_android": {
-            "version_added": false
-          },
-          "safari": {
-            "version_added": false
-          },
-          "safari_ios": {
-            "version_added": false
-          },
-          "samsunginternet_android": {
-            "version_added": "5.0"
+            "version_added": "21"
           },
+          "safari": [
+            {
+              "version_added": "10"
+            },
+            {
+              "version_added": "3",
+              "partial_implementation": true,
+              "notes": "Implemented as an alias for <code>Document</code>."
+            }
+          ],
+          "safari_ios": [
+            {
+              "version_added": "10"
+            },
+            {
+              "version_added": "1",
+              "partial_implementation": true,
+              "notes": "Implemented as an alias for <code>Document</code>."
+            }
+          ],
+          "samsunginternet_android": [
+            {
+              "version_added": "2.0"
+            },
+            {
+              "version_added": "1.0",
+              "partial_implementation": true,
+              "notes": "Implemented as an alias for <code>Document</code>."
+            }
+          ],
           "webview_android": {
-            "version_added": "45"
+            "version_added": "≤37"
```

Comment:
Perhaps we should also make this an array, setting both of the versions to ≤37?

Reply:
The&nbsp;property is&nbsp;no&nbsp;longer an&nbsp;alias in&nbsp;some&nbsp;**≤37**&nbsp;versions&nbsp;though.

Reply:
Ah, if that’s the case, we should have `“version_removed”: “34”` in the compat data above for the statements indicating the alias.

Reply:
There&nbsp;wasn’t **WebView&nbsp;34 for&nbsp;Android**, only&nbsp;**WebView&nbsp;33 for&nbsp;Android**&nbsp;(**4.4.3**) and&nbsp;**WebView&nbsp;37 for&nbsp;Android**.